R/zchunk_L245.water.demand.municipal.R

```diff
@@ -1,253 +0,0 @@
-#' module_water_L245.water.demand.municipal
```

Comment:
Uh...why is this file being removed?

Reply:
I do not have an answer.  Maybe someone is trying to make me crazy by removing random files.  That is the simplest answer.
I made a copy of this chunk from master and put it in my branch to rectify this issue.

Reply:
i bet it was FMS!